### PR TITLE
Setup ci

### PR DESCRIPTION
**Pull request by marcelbuesing**
Also fixed the suggestions. I was skeptical first about all the unspecified in enums but I think the problem with protobuf is anyway that most things are nullable already.
Names of enum values must be unique within the file so i decided it's probably better to use the enum name kind of as a namespace.

### CODE REVIEW

File: lieferemma_api.proto [hunk 1]
```diff
@@ -1,391 +1,408 @@
...
-    // Currency to display
-    Currency currency = 6;
-    // the unit the product is sold in e.g ( Stück, kg, Kasten, m )
-    Unit unit = 7;
-    // The max possible order size for each product ( Defined to not have to create a inventory)
-    uint32 max_order = 8;
+  rpc UpdateDeliveryStatus(UpdateDeliveryStatusRequest) returns (UpdateDeliveryStatusReply);
 }
 
+// ########################################### Producer APIs
```
Comment (ahelhindi) [inline lieferemma_api.proto]: please remove the newline here

File: lieferemma_api.proto [hunk 2]
```diff
@@ -1,391 +1,408 @@
...
+  // Radius of interest e.g. when viewing map
+  // May be ignored by server when exceeding a certain distance
+  int32 radius_in_meter = 2;
 }
 
 enum DeliveryStatus {
-    ON_TOUR = 0;
-    PARKED = 1;
+  // The 0 value is required by protobuf but is not a valid value!
+  DELIVERY_STATUS_UNSPECIFIED = 0;
```
Comment (ahelhindi) [inline lieferemma_api.proto:23]: I do not understand the the reason for using the namespace in the enum.

Comment (ahelhindi) [reply lieferemma_api.proto:23]: Ok looks like it is a problem with protobuf not allowing double usage of values in different enums. we will keep it in that case

File: lieferemma_api.proto (hunk 1 above)
Comment (marcelbuesing) [reply lieferemma_api.proto]: Removed

**Pull request merged**